app/controllers/api/v1/worlds_controller.rb

```diff
@@ -0,0 +1,52 @@
+class Api::V1::WorldsController < ApplicationController
+  before_action :set_world, only: %i[show update destroy]
+
+  # GET /worlds
+  def index
+    @worlds = World.all
+
+    render json: @worlds
+  end
+
+  # GET /worlds/1
+  def show
+    render json: @world
+  end
+
+  # POST /worlds
+  def create
+    @world = World.new(world_params)
+
+    if @world.save
+      render json: @world, status: :created, location: @world
+    else
+      render json: @world.errors, status: :unprocessable_entity
+    end
+  end
+
```

Comment:
- [ ] Kindly Make sure your API is responsible for getting the random greeting word from the DATABASE. Add an action for that here.

Reply:
Hi @andersonlebon as you may see above those lines the controller is calling the API (I left an screenshot) and it's handling the HTTP request. The JSON files are mimics of the database ` @worlds = World.all render json: @worlds`. If you take a look at the App.js component, you'll find that I didn't import any function from greeting component meaning it's all API.  Hope this explains a little bit. 

![Captura de pantalla (92)](https://user-images.githubusercontent.com/103516637/211005903-437fc2cc-0ff8-488b-9511-05e8751e06a4.png)